android/quest/src/gizEir/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="no_found_locations_matching_text">No caregiver found matching text \"%1$s\"</string>
+</resources>
```

Comment:
Close this PR and create a new PR pointing to the main branch with this change in gizEir build variant. @f-odhiambo

Reply:
@ellykits  I might be wrong but this is on the gizEir strings.xml.  I think it's fine there